packages/reveal-latte/tests/Rules/LatteCompleteCheckRule/LatteCompleteCheckRuleTest.php

```diff
@@ -86,7 +86,7 @@ public function provideData(): Iterator
             ],
             ['Call to an undefined method ' . ControlWithHandle::class . '::handleUnknown().', 18],
             [
-                'Parameter #2 $add of method ' . FooPresenter::class . '::renderDefault() expects array|null, string given.',
+                'Parameter #2 $add of method ' . FirstFooPresenter::class . '::renderDefault() expects array|null, string given.',
```

Comment:
This is pretty hard to review for me, as the class has changed and result is different.
Instead it would be better to have a new test for modules, so the original is still tested too :)

Reply:
You're right, I've add it back and add new tests